framework/src/dgkernels/DGKernel.C

```diff
@@ -29,15 +29,12 @@
 #include "libmesh/libmesh_common.h"
 #include "libmesh/quadrature.h"
 
-const BoundaryID DGKernel::InternalBndId = 12345;
```

Comment:
As far as I could tell this member served no purpose. I took it to mean that a developer in the past had premonitions about adding BoundaryRestriction but never got around to it. Before this PR DGKernels were incapable of boundary restriction, so what use would this seemingly miscellaneous BoundaryID have? That was my take. I very well could have missed its importance however.